### DISCUSSION TITLE
Would like help with Minio CPU consumption

### DISCUSSION THREAD

**Discussion by mournblade** [minio/minio#16867] [Q&A] [answered]
Hi, we're running minio on a stand-alone server since a couple of years, which has worked well.
We're running version 2019-10-12T01:39:57Z in our current production environment.
We have a little less than 300 buckets, perhaps 9000 files in total.

We are migrating to new servers, and thought we'd upgrade minio, which i'm no expert on.
I did read up, and noticed something about the mode we're running on was deprecated 
(https://min.io/docs/minio/linux/operations/install-deploy-manage/migrate-fs-gateway.html).

Our goal was to have the absolute minimum amount of hassle, so i surmised that we had to use a version released before
2022-10-29. If we wanted to avoid having to do some sort of data migration process.

So we have installed "2022-10-24T18:35:07Z" and just copied the data dir into the same place on the new machine.

All works fine, functionality wise, with just copying the data dir, however, i notice that minio on the new server uses up significantly more CPU resources.

I was hoping that someone might be able to point me to some reasons to why this might be happening, and if there is something i can do about it. 
I'd rather not install the old version in the new environment if i can avoid it.

My guess, obviously, is that there is some feature running a lot that we don't need or use. 

I poked around a bit, and could it be something with "scanning"? the setting right now is "delay multiplier" 2, max wait 5 and cycle 1m.

I would very much appreciate pointers. Perhaps there is some other, obvious thing I can do.

I attached the /data/.minio_sys/config/config.json if it's useful
[minio_ config_json.txt](https://github.com/minio/minio/files/11037092/minio_.config_json.txt)

Huge thanks in advance.

**Comment by klauspost** [comment, marked as answer]
We don't provide support for older releases and the NAS gateway is deprecated.

Making scanner delay bigger will reduce idle consumption by the same factor. Ie 2 -> 20 will reduce it by a factor of 10.

**Comment by mournblade** [comment]
ok. We're planning to upgrade minio when we get the time, it's just that there's a lot of work understanding and migrating our existing data - right now we must focus on moving to new production servers.

Since all my files are just in a directory on the hard drive, is the scanner really crucial to us (the directory is being  backed up)? I'm not sure what's scanner values would be reasonable for me to set right now.

I know you don't provide support, but perhaps you can just throw a couple of comments my way :)
